libexec/rtld-elf/rtld.h

```diff
@@ -65,6 +65,14 @@
 extern bool ld_compartment_enable;
 
 #define	C18N_ENABLED	ld_compartment_enable
+
+#ifdef __aarch64__
```

Comment:
Should this be defined in an MD header instead?

Reply:
Ah, I think I missed deleting this when disentangling the PR from the CHERI-RISC-V port.